parquet-column/src/main/java/org/apache/parquet/column/statistics/Statistics.java

```diff
@@ -44,10 +45,10 @@ public Statistics() {
    * @param type PrimitiveTypeName type of the column
    * @return instance of a typed statistics class
    */
-  public static Statistics getStatsBasedOnType(PrimitiveTypeName type) {
+  public static Statistics getStatsBasedOnType(PrimitiveTypeName type, BloomFilterOpts opts) {
```

Comment:
this should be a generic statistics collection configuration object. It would contain the BloomFilter options as well.
class StatisticsOpts { BloomFilterOpts bloomFilterOpts; ...
getStatsBasedOnType(PrimitiveTypeName type, StatisticsOpts opts) {